src/app/modules/item/pages/item-log-view/item-log-view.component.ts

```diff
@@ -25,19 +26,20 @@ interface Data {
 export class ItemLogViewComponent implements OnChanges, OnDestroy {
 
   @Input() itemData?: ItemData;
-  @Input() isWatchingGroup = false;
-  @Input() watchingGroupId?: string;
 
   private readonly refresh$ = new Subject<void>();
   private readonly item$ = new ReplaySubject<Item>(1);
-  readonly state$ = this.item$.pipe(
-    distinct(),
-    switchMap((item: Item) => this.getData$(item)),
+  readonly state$ = combineLatest([
+    this.item$.pipe(distinct()),
+    this.sessionService.session$,
```

Comment:
I suppose you can use `this.sessionService.watchedGroup$` directly.

Reply:
Fixed